src/main/java/ip/response/Response.java

```diff
@@ -0,0 +1,39 @@
+package ip.response;
+
+public class Response {
+    // Horizontal line
+    public static void horizontalLine() {
```

Comment:
Perhaps horizontalLine() method can be named such that it is a verb? (e.g. printHorizontalLine())

Reply:
Have addressed this. Thanks for the feedback!